src/callTraceStorage.h

```diff
@@ -40,7 +40,7 @@ struct CallTraceSample {
         return *this;
     }
 
-    bool operator<(const CallTraceSample& other) const {
+    bool operator>(const CallTraceSample& other) const {
         return counter > other.counter;
     }
```

Comment:
I noticed this in the new unit test:

```
    left.counter = 11;
    right.counter = 100;
    ASSERT(left < right); // was failing.
```

A smaller counter value doesn't make the `CallTraceSample` smaller. This operator is used only from `profiler.cpp` while dumping traces to the text output format. I flipped the sort order there instead and kept the operator implementation unsurprising.

This was further tested by dumping traces and comparing them before/after the change.